src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -70,3 +71,36 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
     decode[List[ZuoraErrorResponse]](responseBody).map(_.head)
 
 }
+
+object GetAccountForIdentity {
+
+  case class ZuoraAccountNumber(value: String)
```

Comment:
this should extend AnyVal, but mockito seems to blow up with an NPE if I do that on https://github.com/guardian/support-workers/pull/147/files#diff-94103699f37bee6329145bc9bae91069R73